frontend/riskofbias/constants.js

```diff
@@ -77,6 +77,16 @@ const NA_KEYS = [10, 20],
         26: "Adequate (metric) or Medium confidence (overall)",
         27: "Good (metric) or High confidence (overall)",
     },
+    BIAS_DIRECTION_SIMPLE = {
+        0: "?",
```

Comment:
create a constant `BIAS_DIRECTION_UNKNOWN = 0` (and probably the others too) so that we can refer to named constants instead of integers